.gitignore

```diff
@@ -1,3 +1,4 @@
 coverage/
 node_modules/
 test/output/
+.idea
```

Comment:
Please revert this change, as it is unrelated (you can submit it in a separate PR).